packages/client/hmi-client/src/components/home/tera-card-carousel.vue

```diff
@@ -0,0 +1,202 @@
+<template>
+	<main>
+		<section class="carousel" ref="carouselRef">
+			<div v-if="activeCarouselPage > 1" class="chevron-left" @click="scroll('left')">
+				<i class="pi pi-chevron-left" />
+			</div>
+			<div
+				v-if="activeCarouselPage !== amountOfCardPages"
+				class="chevron-right"
+				@click="scroll('right')"
+			>
+				<i class="pi pi-chevron-right" />
+			</div>
+			<ul v-if="isLoading">
+				<li v-for="i in amountOfSkeletonCards" :key="i">
+					<slot name="skeleton-card" />
+				</li>
+			</ul>
+			<ul v-else ref="cardListRef">
+				<slot name="card-list-items" />
+			</ul>
+		</section>
+		<section v-if="amountOfCardPages > 1" class="page-indicators">
+			<!--page starts counting at 1 because we are looping through a number-->
+			<Button
+				v-for="page in amountOfCardPages"
+				:key="page"
+				icon="pi pi-circle-fill"
+				:active="activeCarouselPage === page"
+				class="page-indicator p-button-icon-only p-button-text p-button-rounded"
+				@click="onCarouselPaginate(page)"
+			/>
+		</section>
+	</main>
+</template>
+
+<script setup lang="ts">
+import { computed, ref, watch, onMounted, onBeforeUnmount } from 'vue';
+import Button from 'primevue/button';
+
+const props = defineProps({
+	isLoading: {
+		type: Boolean,
+		default: false
+	},
+	amountOfCards: {
+		type: Number,
+		default: 0
+	},
+	amountOfSkeletonCards: {
+		type: Number,
+		default: 3
+	}
+});
+
+// If cards end up varying in width cardWidth could be a prop
+const cardWidth = 238; // 17rem
+const rightGapWidth = 21; // 1.5rem
+
+const carouselRef = ref();
+const cardListRef = ref();
+const carouselWidth = ref(0);
+const activeCarouselPage = ref(1);
+
+const amountOfCardsToMove = computed(() =>
+	carouselWidth.value ? Math.floor(carouselWidth.value / (cardWidth + rightGapWidth)) : 3
```

Comment:
Looks like `carouselWidth` will always have a value, so no need for the conditional?